src/Symfony/Component/Security/Core/Authorization/AccessDecisionManager.php

```diff
@@ -160,7 +159,6 @@ private function decideConsensus(TokenInterface $token, array $attributes, $obje
                     break;
 
                 default:
```

Comment:
only if they would request it and use it, we already implemented few fixers only for symfony which are not in use.